### PR TITLE
Proposed changes (see below)

### PR DESCRIPTION
**Pull request by nickwei84**
- switched from using Godep to gvt for managing dependencies
- updated to Go 1.6
- changed "en_US" to "en-us" to be consistent with the goi18n translation file name
- removes updated translations instead of additional translations in the logic for "fixup"
- a couple other changes to work with goi18n

### CODE REVIEW

**Comment by nickwei84** [comment]
@maximilien forgot to run the tests locally. We will update this when fixed.

**Comment by maximilien** [comment]
Hi, Can you fix travis failure? Seems that tests are not passing?

**Comment by nickwei84** [comment]
@maximilien @challiwill
We were working on fixing test failures. Something else might have been prioritized first. Corbin has more context on this.

**Comment by challiwill** [comment]
@maximilien Yes still working on fixing things. Unfortunately got side tracked with some other work. We can remove this PR until it's ready if you would prefer.

**Comment by maximilien** [comment]
@challiwill no problem. Ping me when ready. I'll leave it open. Cheers.

File: integration/merge_strings/d_option_test.go [hunk 1]
```diff
@@ -46,7 +46,7 @@ var _ = Describe("merge-strings -d dirName", func() {
 				)
 			})
 
-			It("creates an en.all.json for which the translation strings order are stable", func() {
+			FIt("creates an en.all.json for which the translation strings order are stable", func() {
```
Comment (maximilien) [inline integration/merge_strings/d_option_test.go]: Why are we `FIt` this test? Ignores the others...

File: integration/merge_strings/d_option_test.go [hunk 2]
```diff
@@ -59,6 +59,7 @@ var _ = Describe("merge-strings -d dirName", func() {
 				Ω(actualBytes).ShouldNot(BeNil())
 
 				Ω(string(expectedBytes)).Should(Equal(string(actualBytes)))
+				// Ω(expectedBytes).Should(MatchJSON(actualBytes))
```
Comment (maximilien) [inline integration/merge_strings/d_option_test.go]: Remove commented code

**Comment by nickwei84** [comment]
Closing until we determine how the CF CLI is going to use its i18n tooling.

**Pull request closed without merging**